selftests: ringbuffers: failing with timeout

Running selftest/ringbuffers we got `[!] ERROR: selftest timeout`:

```bash
libbpfgo/selftest/ringbuffers on 🌱 main [📝🌟] via 🐹 v1.17.1
❯ make run-static
INFO: entering ./selftest/ringbuffers...
make[1]: Entering directory '/home/grg/code/libbpfgo/selftest/ringbuffers'
make -C /home/grg/code/libbpfgo libbpfgo-static
make[2]: Entering directory '/home/grg/code/libbpfgo'
CC=clang \
  CGO_CFLAGS="-I/home/grg/code/libbpfgo/output" \
  CGO_LDFLAGS="-lelf -lz /home/grg/code/libbpfgo/output/libbpf.a" \
  GOOS=linux GOARCH=amd64 \
  go build \
  -tags netgo -ldflags '-w -extldflags "-static"' \
  .
make[2]: Leaving directory '/home/grg/code/libbpfgo'
CC=clang \
  CGO_CFLAGS="-I/home/grg/code/libbpfgo/output" \
  CGO_LDFLAGS="-lelf -lz /home/grg/code/libbpfgo/output/libbpf.a" \
  GOOS=linux GOARCH=amd64 \
  go build \
  -tags netgo -ldflags '-w -extldflags "-static"' \
  -o main-static ./main.go
sudo ./run.sh main-static
[!] ERROR: selftest timeout
make[1]: *** [Makefile:85: run-static] Error 3
```

--- a/selftest/ringbuffers/main.go
+++ b/selftest/ringbuffers/main.go
@@ -5,6 +5,7 @@ import "C"
 import (
 	"os"
 	"syscall"
+	"time"
 
 	"encoding/binary"
 	"fmt"
@@ -69,6 +70,7 @@ func main() {
 	go func() {
 		for {
 			syscall.Mmap(999, 999, 999, 1, 1)
+			time.Sleep(time.Second / 2)
 		}
 	}()
 recvLoop:

